### DISCUSSION TITLE
llamacpp-for-kobold-1.0.6-beta

### DISCUSSION THREAD

**Discussion by LostRuins** [LostRuins/koboldcpp#8] [General]
**llamacpp-for-kobold-1.0.6-beta**

- This is an experimental release containing new integrations for OpenBLAS, which should increase initial prompt processing speed on compatible systems by over 2 times!
- Updated Embedded Kobold Lite with the latest version which supports pseudo token streaming. This should make the UI feel much more responsive during prompt generation.
- Switched to argparse, you can view all command line flags with `llamacpp-for-kobold.exe --help`
- To disable OpenBLAS, you can run it with `--noblas`. Please tell me if you have issues with it, and include which specific OS and platform.

To use, download and run the **llamacpp-for-kobold.exe**
Alternatively, drag and drop a compatible quantized model for llamacpp on top of the .exe, or run it and manually select the model in the popup dialog.

and then once loaded, you can connect like this (or use the full koboldai client):
**http://localhost:5001**

<hr /><em>This discussion was created from the release <a href='https://github.com/LostRuins/llamacpp-for-kobold/releases/tag/'>llamacpp-for-kobold-1.0.6-beta</a>.</em>

**Comment by aphexddb** [comment]
Virustotal flags [llamacpp-for-kobold.exe](https://www.virustotal.com/gui/file/1a3949226d498c1ffa55e0a43dd9a2b2ce49c187b4d6b3d8ce4e48c6d477ec73) as malicious

**Comment by LostRuins** [comment]
That is concerning. It is a standard pyinstaller install. You can build everything yourself from source using the makefile and the make_pyinstaller.bat file. I have no idea why it would be flagged.
